### DISCUSSION TITLE
Should we support x86 (32 bit archintecture) ?

### DISCUSSION THREAD

**Discussion by rabbit-aaron** [OctoPrint/docker#218] [Ideas]
Hey guys, awesome project!

I'm running this on my really old laptop, it has an Atom N270 processor which is x86 (or i686), I was able to build the image by changing 

```FROM ubuntu AS s6build```
to
```FROM debian AS s6build```

and add

```i386) ARCH='x86';; \``` 

then it builds and runs fine on my laptop.

For newer versions of s6, we'd need to use 

```i386) ARCH='i686';; \```

**Comment by cisasmendi** [comment]
You are right since it does not support x86 if the modification is simple. Thanks I was able to make it work
